terraform/Makefile

```diff
@@ -17,19 +21,24 @@ terraform/backend: checks
 	fi
 	@echo "Creating Terraform state file in ./${TERRAFORM_PROJECT}/terraform.backend.tf from template in ./${TERRAFORM_PROJECT}/terraform.backend.tf.dist"
 	@sed "s/PR_NUMBER/${PR_NUMBER}/g" "./${TERRAFORM_PROJECT}/terraform.backend.tf.dist" > "./${TERRAFORM_PROJECT}/terraform.backend.tf"
-	cat "./${TERRAFORM_PROJECT}/terraform.backend.tf"
+
+# Exports environment variables that are accessed by the launched Terraform project
+.PHONY: terraform/vars
+terraform/vars:
+	echo "pr_number = \"${PR_NUMBER}\"" >> "./${TERRAFORM_PROJECT}/variables.tfvars"
```

Comment:
The `pr_number` is now passed as a variable when executing the Terraform modules, so that the created resources will include it in its naming.